src/views/SystemProfiles/index.tsx

```diff
@@ -1,13 +1,158 @@
+  {
+    id: '5',
+    name: 'System5',
+    acronym: 'SYS5',
+    slug: 'system5',
+    link: '/system/system5'
+  }
+];
+
+type SystemProfilesProps = {
+  auth: any;
+};
+
+export const SystemProfiles = ({ auth }: SystemProfilesProps) => {
+  const onSearch = () => {};
+  const getSuggestionValue = (suggestion: any): string => suggestion.name;
+  const renderSuggestion = (suggestion: any): string => suggestion.name;
+  const intakes = useSelector(
+    (state: AppState) => state.systemIntakes.systemIntakes
+  );
+  const searchResults = useSelector(
+    (state: AppState) => state.search.allSystemShorts
+  );
+  const [timeStamp, setTimeStamp] = useState('');
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    const getSystemIntakes = async (): Promise<void> => {
+      dispatch(fetchSystemIntakes(await auth.getAccessToken()));
+      await setTimeStamp(
+        DateTime.local().toLocaleString(DateTime.DATETIME_MED)
+      );
+    };
+    getSystemIntakes();
+  }, [auth, dispatch]);
+
+  useEffect(() => {
+    const fetchSystemShorts = async (): Promise<void> => {
+      dispatch(getAllSystemShorts(await auth.getAccessToken()));
+    };
+    fetchSystemShorts();
+  }, [auth, dispatch]);
+
+  const getStatusNotification = (status: string) => {
+    switch (status) {
+      case 'SUBMITTED':
+        return 'Status: The form has been submitted and is being reviewed.';
+      case 'REVIEWED':
+        return 'Status: Request Form has been reviewed. Prepare your business case.';
+      default:
+        return '';
+    }
+  };
+
+  const getStatusButton = (status: string) => {
+    switch (status) {
+      case 'SUBMITTED':
+        return (
+          <Button type="button" unstyled>
+            View submitted request form
+          </Button>
+        );
+      case 'REVIEWED':
+        return <Button type="button">Begin Business Case</Button>;
+      default:
+        return '';
+    }
+  };
 
-const SystemProfiles = () => {
   return (
     <div>
-      <Header />
-      <h1>All System Profiles List Page</h1>
+      <Header>
+        <div className="system-profile__search-bar">
+          <SearchBar
+            name="System search"
+            onSearch={onSearch}
+            getSuggestionValue={getSuggestionValue}
+            renderSuggestion={renderSuggestion}
+            results={searchResults}
+          />
+        </div>
+
+        {mockSystems.length > 0 && (
+          <div className="system-profile__secondary-nav-wrapper">
+            <SecondaryNav
+              secondaryNavList={mockSystems.slice(0, 10)}
+              activeNavItem="all"
+            />
+          </div>
+        )}
+      </Header>
+      <div className="grid-container">
+        <UpcomingActions timestamp={timeStamp}>
+          {intakes &&
+            intakes
+              .filter(intake => ['SUBMITTED, REVIEWED'].includes(intake.status))
+              .map(intake => {
+                return (
+                  <Fragment key={intake.id}>
```

Comment:
I don't think we need a `Fragment` here. You can stick the `key` directly on `ActionBanner`.